packages/core-components/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -55,6 +55,9 @@ const useStyles = makeStyles(
     headerAvatar: {},
     headerAction: {},
     headerContent: {},
+    subheader: {
+      float: 'left',
```

Comment:
Just because we don't use this much and it's scary, do you think we can achieve the same thing with flex instead? Fine to keep as is if that'd cause trouble tho